alacritty/src/config/bindings.rs

```diff
@@ -1213,6 +1214,21 @@ impl<'a> Deserialize<'a> for KeyBinding {
     }
 }
 
+#[derive(SerdeReplace, Debug, Copy, Clone, Eq, PartialEq)]
+pub struct ViMotionWrapper(ViMotion);
```

Comment:
Not really a fan of naming a variable after what it is, rather than *why* it is.

Calling it just `ViMotion` would be fine. If you wanted to change the name a more descriptive one would be `SerdeViMotion` which communicates *why* it is wrapped.